test/prometheus-scrape.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+# This is a test suite for testing basic log processing
+# functionality and existance of duplicate records when journal rolls over
+
+source "$(dirname "${BASH_SOURCE[0]}" )/../hack/lib/init.sh"
+source "${OS_O_A_L_DIR}/hack/testing/util.sh"
+os::util::environment::use_sudo
+
+os::test::junit::declare_suite_start "test/prometheus-scrape"
```

Comment:
If prometheus is on by default, then you could add a check to https://github.com/openshift/origin-aggregated-logging/blob/master/test/cluster/functionality.sh#L140 and skip this test

Reply:
ack ack. I plan on removing this test actually eventually. I admittedly am still having issues building locally so I can't test if there are additional `<source>` configs we need to add as part of making metrics available at this endpoint.

It may be useful to add a functionality test though just as a sanity check that the plugin is still working.